### PR TITLE
Add support for both tag and hash for images. Hash will be used if no tag is set

### PR DESCRIPTION
**Pull request by Schreglmann**

### CODE REVIEW

File: charts/comet-site-v8/templates/build-cron-job.yaml [hunk 1]
```diff
@@ -38,7 +38,13 @@ spec:
           serviceAccountName: {{ include "comet-site.fullname" . }}
           containers:
             - name: {{ .Chart.Name }}-build
+              securityContext:
+                {{- toYaml .Values.builder.securityContext | nindent 16 }}
```
Comment (dkarnutsch) [inline charts/comet-site-v8/templates/build-cron-job.yaml:42]: Not part of this PR

**Review by dkarnutsch — CHANGES_REQUESTED** [review]

File: charts/comet-site-v8/templates/build-cron-job.yaml (hunk 1 above)
Comment (Schreglmann) [reply charts/comet-site-v8/templates/build-cron-job.yaml]: moved to this PR: https://github.com/vivid-planet/comet-charts/pull/39

**Review by dkarnutsch — APPROVED** [review]
